packages/bridge-ui/src/components/buttons/SelectToken.svelte

```diff
@@ -34,29 +41,25 @@
     }
   }
 
-  let showAddressField = false;
-
-  let loading = false;
-
   async function addERC20(event: SubmitEvent) {
     try {
       loading = true;
       const eventTarget = event.target as HTMLFormElement;
       const { customTokenAddress } = eventTarget;
       const tokenAddress = customTokenAddress.value;
-      if(!ethers.utils.isAddress(tokenAddress)) {
+      if (!ethers.utils.isAddress(tokenAddress)) {
         throw new Error();
```

Comment:
should we throw a valuable error to display to the user?

Reply:
Yeah, happens here is that this whole logic is within a try-catch. In the catch we present a toast saying: 'Not a valid ERC20 address'. The problem I see is that many things might go wrong here, not just an invalid ERC20 address. I'll change this error handling a bit.